trustkit/src/main/java/com/datatheorem/android/trustkit/config/TrustKitConfigurationParser.java

```diff
@@ -4,6 +4,8 @@
 import android.content.Context;
 import androidx.annotation.NonNull;
 import android.text.TextUtils;
+import android.util.Log;
```

Comment:
Unused import.

Reply:
Good catch will fix it probably tomorrow

Reply:
Should be good :)